### PR TITLE
Feature/credit line closing info

### PR DESCRIPTION
**Pull request by hhemanth**
@ouranos 
Refactored code to handle Closing transaction of CreditLine, not just CreditCard

### CODE REVIEW

**Review by xiuzhong — APPROVED** [review]
LGTM

**Comment by ouranos** [comment]
Changed the base to `staging-v2` to facilitate review

File: spec/fixtures/credit_line_closing_info.ofx [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+   <BANKACCTFROM>
+    <BANKID>111777882</BANKID>
+    <ACCTID>4307754</ACCTID>
+    <ACCTTYPE>CREDITLINE</ACCTTYPE>
+   </BANKACCTFROM>
+  </STMTENDRQ>
+ </STMTENDTRNRQ>
+</OFX>
+
+        Response:-
```
Comment (ouranos) [inline spec/fixtures/credit_line_closing_info.ofx]: Does this file contains 2 xml files?

**Review by ouranos — APPROVED** [review]

File: spec/fixtures/credit_line_closing_info.ofx (hunk 1 above)
Comment (hhemanth) [reply spec/fixtures/credit_line_closing_info.ofx]: It had both request and response, removed request xml, as it is not needed

**Comment by ouranos** [comment]
@hhemanth This got automatically merged because the base if staging-v2. Opened #5 against the `maestrano` branch for production release

**Pull request merged**